From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mukul Sikka <mukul.sikka@broadcom.com>
Date: Fri, 22 May 2026 00:00:00 +0000
Subject: [PATCH] fix: bump flux to 0.194.5 and immutable to 0.4.3

Bump github.com/influxdata/flux to v0.194.5 and
github.com/benbjohnson/immutable to v0.4.3 to match the generic
SortedMap API used by the flux-0.194.5-go1.23.patch.

---
 go.mod | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/go.mod b/go.mod
index abcdef1..abcdef2 100644
--- a/go.mod
+++ b/go.mod
@@ -23,7 +23,7 @@ require (
 	github.com/gorhill/cronexpr v0.0.0-20180427100037-88b0669f7d75
 	github.com/h2non/gock v1.2.0
 	github.com/influxdata/cron v0.0.0-20201006132531-4bb0a200dcbe
-	github.com/influxdata/flux v0.191.0
+	github.com/influxdata/flux v0.194.5
 	github.com/influxdata/httprouter v1.3.1-0.20191122104820-ee83e2772f69
 	github.com/influxdata/influx-cli/v2 v2.0.0-20210526124422-63da8eccbdb7
 	github.com/influxdata/influxdb v1.9.6
@@ -115,7 +115,7 @@ require (
 	github.com/aws/aws-sdk-go-v2/service/internal/s3shared v1.14.0 // indirect
 	github.com/aws/aws-sdk-go-v2/service/s3 v1.31.0 // indirect
 	github.com/aws/smithy-go v1.24.2 // indirect
-	github.com/benbjohnson/immutable v0.3.0 // indirect
+	github.com/benbjohnson/immutable v0.4.3 // indirect
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/bonitoo-io/go-sql-bigquery v0.3.4-1.4.0 // indirect
 	github.com/cespare/xxhash/v2 v2.2.0 // indirect
-- 
2.49.0
